client/my-sites/media/main.jsx

```diff
@@ -268,10 +272,28 @@ class Media extends Component {
 		MediaActions.delete( site.ID, selected );
 	};
 
+	getAnalyticsPath = () => {
+		const { filter, selectedSite } = this.props;
+		let analyticsPath = '/media';
+
+		if ( filter ) {
+			analyticsPath += `/${ filter }`;
+		}
+
+		if ( selectedSite && selectedSite.ID ) {
+			analyticsPath += '/:site';
+		}
+
+		return analyticsPath;
+	};
+
 	render() {
+		const { translate } = this.props;
 		const site = this.props.selectedSite;
 		return (
 			<div ref="container" className="main main-column media" role="main">
+				<DocumentHead title={ translate( 'Media' ) } />
```

Comment:
Note: I noticed that this is not setting the title correctly when switching filters. This is not good to merge until that is resolved.

Reply:
From the old code wasn't this the case before? We can try and tidy that up in an additional janitorial if that's the case.

Reply:
You're right there is an odd regression, where the page title goes back to the site name.

Reply:
There's similar bad behavior with switching between comment filters. Maybe keep the old `setTitle` usage, and we can look into a separate PR for fixing DocumentHead